src/merge_sort.py

```diff
@@ -0,0 +1,64 @@
+"""A merge sort function."""
+from insertion_sort import existing_set_insertion_sort as esis
+
+from insertion_sort import insertion_sort
+
+
+def merge_sort(numbers):
+    """Sort a iterable of numbers into min first via merge sort."""
+    final = []
+    compare = []
+    temp = []
+    hold_extra = []
+    if len(numbers) == 1:
+        return numbers
+    elif len(numbers) == 2:
+        if numbers[0] > numbers[1]:
+            numbers[0], numbers[1] = numbers[1], numbers[0]
+            return numbers
```

Comment:
It's strange how if I give this a len 1 or 2 list to sort, it will sort my list in place, but if it's longer it won't.